### PR TITLE
Migrate workflow to mirror Copr RPMs into Cloudsmith archive to Github Actions

### PR DESCRIPTION
**Pull request by glebashnik**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

## What

Adds a new workflow to mover Copr RPMs to archive.
The workflow is triggered either manually or on a daily schedule.

## Why

Part of the process of retiring Screwdriver as CI/CD runner.

## Additional Info

The corresponding Screwdriver pipeline is not removed since we don't have a dashboard for Github Actions yet.

See successful test run: 
https://github.com/vespa-engine/vespa/actions/runs/10385493605

### CODE REVIEW

File: .github/workflows/mirror-copr-rpms-to-archive.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        run: |
+          dnf install -y dnf-plugins-core jq
+
+      - name: Publish to x86-64 mirror
+        run: |
+          .github/scripts/publish-unpublished-rpms-to-archive.sh x86_64
+
+      - name: Publish to aarch64 mirror
+        run: |
+          .github/scripts/publish-unpublished-rpms-to-archive.sh aarch64
```
Comment (esolitos) [inline .github/workflows/mirror-copr-rpms-to-archive.yml:47]: In this case I would use the [matrix strategy](https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/choosing-what-your-workflow-does/using-a-matrix-for-your-jobs), for example:

```yml

jobs:
  my-job:
    strategy:
      matrix:
        arch: ['x86_64', 'aarch64']
    steps:
      [...]
      - name: Publish to aarch64 mirror
        env:
          CPU_ARCH: ${{ matrix.arch }}
        run: |
          .github/scripts/publish-unpublished-rpms-to-archive.sh $CPU_ARCH

Comment (glebashnik) [reply .github/workflows/mirror-copr-rpms-to-archive.yml:47]: Makes sense. Refactored.

**Review by esolitos — CHANGES_REQUESTED** [review]
Looks good, but let's also remove the job from Screwdriver as well a the scripts you copied from /screwdriver.

**Comment by glebashnik** [comment]
Removed the corresponding screwdriver pipeline.

**Comment by glebashnik** [comment]
Removed the corresponding screwdriver pipeline.

**Review by esolitos — APPROVED** [review]

**Comment by aressem** [comment]
For changes like this it would be nice to git move the scripts instead of copying and adding as new files. This would retain the history of the files.

**Pull request merged**